src/components/UserProfile/UserProfile.jsx

```diff
@@ -126,6 +122,57 @@ const UserProfile = props => {
     }
   };
 
+  const getWeeklySummary = async userId => {
+    try {
+      setSummarySelected('');
+      setSubmittedSummary(false);
+      const response = await axios.get(ENDPOINTS.USER_PROFILE(userId));
+      const user = response.data;
+      let summaries = user.weeklySummaries;
+      if (summaries && Array.isArray(summaries) && summaries[0] && summaries[0].summary) {
+        setSummarySelected(summaries[0].summary);
+        setSubmittedSummary(true);
```

Comment:
I checked the bug list. It requires displaying past three week's summaries. Please check it out.
![summary](https://user-images.githubusercontent.com/98206107/208271898-e173c04f-9c00-470d-ab5d-83c5ee3e0733.JPG)